bwa.wdl

```diff
@@ -114,3 +114,35 @@ struct BwaIndex {
     File fastaFile
     Array[File] indexFiles
 }
+
+task Index {
+    input {
+        File fasta
+    }
+    String indexedFile = basename(fasta)
+
+    command {
+        set -e
+        cp ~{fasta} ~{indexedFile}
+        bwa index ~{indexedFile}
+    }
+
+    output {
+        BwaIndex index = object {
+            fastaFile: indexedFile,
+            indexFiles: [
+                indexedFile + ".amb",
+                indexedFile + ".ann",
+                indexedFile + ".bwt",
+                indexedFile + ".pac",
+                indexedFile + ".sa"
+            ]
+        }
+    }
+
+    runtime {
+        docker: "quay.io/biocontainers/bwa:0.7.17--hed695b0_7"
```

Comment:
Given that the bwa task's docker image is configurable, maybe this should be as well